src/fixate/core/discover.py

```diff
@@ -57,27 +54,13 @@ def discover_sub_classes(sub_class_match):
 def _classes_in_module(module, path_to_module):
     try:
         __import__(module)
-        # importlib.machinery.SourceFileLoader(module, path_to_module).load_module()
     except Exception as e:
-        pass
-        # print(e)
-        # print("Import Error: {}".format(module))
-    else:
-        mod = sys.modules[module]
-        return inspect.getmembers(mod, inspect.isclass)
-    return []
-
+        return []
```

Comment:
Doesn't look like this (and discover_classes above) are used anymore. Can we delete?

Reply:
Yeah, that was used before i had DRIVER_LIST